Fix critical bug in Team Config

Clicking on Team in Config generates error modal. Here's console:

index-GwjGWeVG.js:9 Error: A <Select.Item /> must have a value prop that is not an empty string. This is because the Select value can be set to an empty string to clear the selection and show the placeholder.
    at index-GwjGWeVG.js:60:100455
    at Gf (index-GwjGWeVG.js:9:48227)
    at db (index-GwjGWeVG.js:9:67748)
    at _b (index-GwjGWeVG.js:9:83418)
    at Jb (index-GwjGWeVG.js:9:117221)
    at HA (index-GwjGWeVG.js:9:116265)
    at Mh (index-GwjGWeVG.js:9:116095)
    at Yb (index-GwjGWeVG.js:9:112878)
    at h0 (index-GwjGWeVG.js:9:124830)
    at Wi (index-GwjGWeVG.js:9:123378)
ob @ index-GwjGWeVG.js:9
index-GwjGWeVG.js:61 ErrorBoundary caught an error: Error: A <Select.Item /> must have a value prop that is not an empty string. This is because the Select value can be set to an empty string to clear the selection and show the placeholder.
    at index-GwjGWeVG.js:60:100455
    at Gf (index-GwjGWeVG.js:9:48227)
    at db (index-GwjGWeVG.js:9:67748)
    at _b (index-GwjGWeVG.js:9:83418)
    at Jb (index-GwjGWeVG.js:9:117221)
    at HA (index-GwjGWeVG.js:9:116265)
    at Mh (index-GwjGWeVG.js:9:116095)
    at Yb (index-GwjGWeVG.js:9:112878)
    at h0 (index-GwjGWeVG.js:9:124830)
    at Wi (index-GwjGWeVG.js:9:123378) Object
componentDidCatch @ index-GwjGWeVG.js:61

--- a/src/components/LinkAccountSection.tsx
+++ b/src/components/LinkAccountSection.tsx
@@ -46,13 +46,16 @@ export const LinkAccountSection = memo(function LinkAccountSection({
   const [isClaiming, setIsClaiming] = useState(false);
   const [isSavingPreference, setIsSavingPreference] = useState(false);
 
+  // Sentinel value for "None" option since Radix Select requires non-empty string values
+  const NONE_VALUE = '__none__';
+
   const handlePreferredInstrumentChange = async (value: string) => {
     if (!linkedMember) return;
 
     setIsSavingPreference(true);
     try {
-      // value of '' means "None" was selected, convert to null
-      const preferredInstrument = value === '' ? null : value;
+      // NONE_VALUE means "None" was selected, convert to null
+      const preferredInstrument = value === NONE_VALUE ? null : value;
       onUpdateMember({
         ...linkedMember,
         preferredInstrument,
@@ -166,7 +169,7 @@ export const LinkAccountSection = memo(function LinkAccountSection({
                 Preferred Instrument
               </Label>
               <Select
-                value={linkedMember.preferredInstrument || ''}
+                value={linkedMember.preferredInstrument || NONE_VALUE}
                 onValueChange={handlePreferredInstrumentChange}
                 disabled={isSavingPreference}
               >
@@ -174,7 +177,7 @@ export const LinkAccountSection = memo(function LinkAccountSection({
                   <SelectValue placeholder="Select your instrument..." />
                 </SelectTrigger>
                 <SelectContent>
-                  <SelectItem value="">None</SelectItem>
+                  <SelectItem value={NONE_VALUE}>None</SelectItem>
                   {availableRoles.map(role => (
                     <SelectItem key={role} value={role}>
                       {role}

